htrc/__main__.py

```diff
@@ -32,6 +32,8 @@ def download_parser(parser=None):
         help="remove folder if exists")
     parser.add_argument("-o", "--output", help="output directory",
         default='/media/secure_volume/workset/')
+    parser.add_argument("-c", "--concatenate",
```

Comment:
I would add `action="store_true"` and remove `default=False`. Otherwise it will expect an argument like `-c value` to store in the `args.concatenate` action https://docs.python.org/3/library/argparse.html#action

Reply:
(code looks fine! just a usage comment)

one action/use-case question - perhaps concatenate should be the default and page retrieval should be the flag? In which case, maybe `--pages` could be the flag, `dest="concatenate"` would mean none of the rest of the code needs to change, and `action="store_false"` would set the default to `True` and when the flag is there, set it to`False`.

Reply:
Yes, you are correct. I'll remove default=False and add action. I think page retrieval should be default and concatenation is an option.